### PR TITLE
feat: add loadFilteredPolicy() support

### PR DESCRIPTION
**Pull request by rogersgt**
Fix: https://github.com/node-casbin/prisma-adapter/issues/49

### CODE REVIEW

**Comment by hsluoyz** [comment]
@rogersgt plz fix all 3:

image

**Comment by rogersgt** [comment]
> @rogersgt plz fix all 3:
> 
> image

Done!

File: package-lock.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {
   "name": "casbin-prisma-adapter",
-  "version": "1.1.3",
+  "version": "1.2.0",
```
Comment (hsluoyz) [inline package-lock.json:3]: I don't think you should update versions in this file manually

File: src/adapter.ts [hunk 2]
```diff
@@ -29,6 +29,38 @@ export class PrismaAdapter implements Adapter {
     }
   }
```
Comment (hsluoyz) [inline src/adapter.ts:31]: missed filter related stuff, search "filtered" in: https://github.com/node-casbin/sequelize-adapter/pull/64/files

**Review by hsluoyz — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
Comment (rogersgt) [reply package-lock.json:3]: It changed when I ran `npm i`.. I'll take another look. Thanks!

**Comment by hsluoyz** [comment]
@rogersgt

**Comment by rogersgt** [comment]
@hsluoyz sorry for the delay.. I included the `isFiltered` property. Anything else I should include?

**Review by hsluoyz — APPROVED** [review]

**Pull request merged**